src/test/java/com/artipie/rpm/misc/StorageLockerTest.java

```diff
@@ -93,4 +94,27 @@ void addsLockAndThrowsExceptionOnSecondLock() {
         );
     }
 
+    @Test
+    void addsLockAndThrowsExceptionOnSecondLockWithRootKey() {
+        final Storage sto = new InMemoryStorage();
+        final StorageLock locker = new StorageLock(sto, Key.ROOT);
+        locker.lock().join();
+        MatcherAssert.assertThat(
+            Assertions.assertThrows(
+                ExecutionException.class,
+                () -> locker.lock().get()
+            ).getCause().getMessage(),
+            new StringContains(
+                String.format(
+                    "Repository '%s' is already being updated, %d locks found.", "", 1
```

Comment:
@olenagerasimova instead of using `String.format` here with the params being literals, just specify the expected literal string.